packages/amplify_authenticator/lib/src/mixins/authenticator_radio_field.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:amplify_authenticator/src/widgets/form_field.dart';
+
+import 'package:flutter/material.dart';
+
+mixin AuthenticatorRadioField<FieldType, FieldValue,
+        T extends AuthenticatorFormField<FieldType, FieldValue, T>>
+    on AuthenticatorFormFieldState<FieldType, FieldValue, T> {
+  void initState() {
+    selectionValue = widget.inputconfig!.selections[0].value as FieldValue;
+    super.initState();
+  }
+
+  @override
+  Widget buildFormField(BuildContext context) {
+    final inputResolver = stringResolver.inputs;
+    onChanged!(selectionValue!);
+    // return Text('Hi');
+    return Column(
+      children: <Widget>[
```

Comment:
*nit:* I guess this is really just preference, but this is easier to understand imo

```dart
children: widget.inputconfig!.selections.map((selection) {
  return ListTile(...)
})
```

Reply:
Actually gonna disagree on this one, I think list comprehensions are nicer. In your example, you gotta add a `.toList()` at the end. Always seemed a bit clunky to me, personally.